src/Middleware/SubstituteBindings.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace LaravelDoctrine\ORM\Middleware;
+
+use Closure;
+use Doctrine\Common\Persistence\Proxy;
+use Illuminate\Contracts\Routing\Registrar;
+use Illuminate\Routing\Middleware\SubstituteBindings as IllumintateSubstituteBindings;
+use LaravelDoctrine\ORM\IlluminateRegistry;
+use ReflectionFunction;
+use ReflectionMethod;
+
+class SubstituteBindings extends IllumintateSubstituteBindings
```

Comment:
Well in the handle method the parent router property is called to add the eloquent substitute bindings. This allows the user to get rid of the default SubstituteBindings middleware. Other than that it is not necessary to extend the original middleware.

I guess we could just have both middleware and just add this middleware to add doctrine support.